core/ast/import.go

```diff
@@ -73,3 +74,11 @@ func (i *ImportNode) String() string {
 	return fmt.Sprintf("ImportNode{ModuleName: %s, ModuleItem: %s, ModuleAlias: %s, WildcardImport: %t}",
 		i.ModuleName(), i.ModuleItem(), i.ModuleAlias(), i.IsWildcardImport())
 }
+
+type ImportJsonString string
```

Comment:
We are introducing a new `core` concept `ImportJsonString` just for testing purpose. Is this really required? Can we not assert against the actual content of the node since we are anyway exposing its member attributes like `ModuleName`, `ModuleItem` etc.

Reply:
`ImportNode` consists of fields which are actually sitter nodes and not just plain strings, recreating these will be tedious for testing purposes

Reply:
Ref: https://github.com/safedep/code/pull/10#discussion_r1914555932